lib/cluster.js

```diff
@@ -178,12 +178,12 @@ function addListeners() {
 }
 
 // Respond to master's requests for worker's local metrics.
-process.on('message', message => {
+process.on('message', async message => {
```

Comment:
Is it safe to make this async?

Reply:
I think so, but I wasn't handling the error. Just revised the commit and switched to a Promise.